components/Favorites.tsx

```diff
@@ -0,0 +1,25 @@
+import { useMovies } from "../stores/moviesStore";
+import "./Favorites.css";
+
+export default function Favorites() {
+  const favorites = useMovies((state) => state.favorites);
+
+  const favoritesList = favorites.map((movie, i) => (
+    <div key={i} className="favorite-movie-title">
+      {i + 1}. {movie.Title}
```

Comment:
always avoid to use index as a key in react
https://medium.com/@sriweb/avoid-using-array-index-as-a-key-in-react-components-1978799ed94b